### PR TITLE
docs: add a new page on 'Proactive Security: Catching Vulnerabilities Early in SDLC'

### PR DESCRIPTION
**Pull request by CalebAbhulimhen**
This change adds a new page to the OWASP community pages. 

It touches on the importance of code security awareness in the early stages of the development cycle; highlighting both the technical and business gains.

It also recommends different options/tools in which software teams can adapt this security-conscious culture to their development process.

### CODE REVIEW

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+#### The Importance of Early Detection
+
+Catching security vulnerabilities early in the development lifecycle—during coding, code review, or testing phases—can drastically reduce the risk of these flaws reaching production. This proactive approach to security is akin to a first line of defense, preventing potential attackers from exploiting known vulnerabilities. By identifying and addressing security issues before the code is deployed, developers can mitigate risks more efficiently, reducing the cost and complexity of fixes later in the development cycle.
+
+Integrating security into the code review process also aligns with the principles of _*Shift Left*_ security, where security practices are moved earlier in the development process. By doing so, security becomes a shared responsibility among all team members, not just a task for dedicated security teams. This collaborative approach ensures that security is not an afterthought but a core aspect of the software development process.
```
Comment (kingthorin) [inline pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md:20]: I don't think `_* text *_` is a supported GH Markdown tag. It displays simply italic.
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+By adopting these practices, developers and organizations can move towards a more secure future, where security is not just a final checkpoint but a continuous, integrated process from the first line of code.
+
+#### More info
+
+* [Free Open Source SAST Tools](__https://owasp.org/www-community/Free_for_Open_Source_Application_Security_Tools__)
+
+* [DAST Tools](__https://owasp.org/www-community/Vulnerability_Scanning_Tools__)
+
+* [Overview on Shift-Left Security Strategy](__https://www.fortinet.com/resources/cyberglossary/shift-left-security__)
```
Comment (kingthorin) [inline pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md:58]: These links are invalid due to the included double underscores.

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Moreover, regulatory frameworks such as the General Data Protection Regulation (GDPR) and the California Consumer Privacy Act (CCPA) impose strict requirements on data security. Failing to comply with these regulations can result in hefty fines. Integrating security into the development process ensures that code is not only secure but also compliant with these regulations from the beginning stages of the SDLC.
+
+#### Conclusion
+
+Incorporating security checks into the development and code review process is a proactive approach to safeguarding applications against potential threats. By catching vulnerabilities early, organizations can mitigate risks, reduce costs, and foster a culture of security awareness among developers. As the security landscape continues to evolve, integrating security into every phase of the development lifecycle will be key to staying ahead of potential threats and ensuring the safety and integrity of applications in production.
+
+By adopting these practices, developers and organizations can move towards a more secure future, where security is not just a final checkpoint but a continuous, integrated process from the first line of code.
+
+#### More info
```
Comment (kingthorin) [inline pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md:52]: Title caps
> More Info

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md (hunk 2 above)
Comment (kingthorin) [reply pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md:58]: image

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md (hunk 1 above)
Comment (CalebAbhulimhen) [reply pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md]: Oh true, thanks for the advice!

File: pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md (hunk 2 above)
Comment (CalebAbhulimhen) [reply pages/Proactive_Security_Catching_Vulnerabilities_Early_in_SDLC.md]: Thanks for catching that! I've updated the links.

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**